andela/ah-scorpion
# #159053985 User Profile

**What does this PR do?**
Adds the functionality to view the profile of a user, given the username

**Description of Task to be completed?**
1. Add **'bio'** and **'image'** fields in the User model.
2. Modify LoginSerializer's validate method to return the User object of the validated user
3. Modify UserSerializer to include **'bio'** and **'image'** fields
4. Modify register and login route to include **'bio'** and **'image'** in response
5. Add tests for the profiles app
6. Add route for viewing a user's profile with no authentication required

**How can this be manually tested?**
- Clone the repository from [here](https://github.com/andela/ah-scorpion/tree/ft-User-Profile-%23159053985 and check out the branch `ft-User-Profile-#159053985`
- Activate your virtual environment and use the `.env-sample` to set the environment variables.
- Use postman to test these endpoints.
  `GET api/profiles/<str:username>` for viewing a user's profile
It should return the user's profile if the user exists, and 404 otherwise

**What are the relevant pivotal tracker stories?**
[#159053985](https://www.pivotaltracker.com/story/show/159053985)
